datalad/distribution/create_sibling.py

```diff
@@ -379,9 +380,14 @@ def __call__(sshurl, name=None, target_dir=None,
         # dataset instances
         datasets = {p: Dataset(p) for p in content_by_ds}
 
+        # make sure dependencies are valid
+        for d in datasets.values():
+            _check_deps(d.repo, publish_depends)
```

Comment:
`publish_depends` sounds like a property of each particular dataset.  I could have some datasets depending on some remotes and some which don't.  Here, testing is done of all resolved datasets against the same set of "publish_depends"... shouldn't publish_depends be stored in the datalad's config, probably not persistent (thus within .git/config) for the dataset?